pkg/testhelper/testhelper.go

```diff
@@ -60,42 +59,35 @@ const (
 	PodType              = "Pod"
 	ContainerType        = "Container"
 	ContainerProcessType = "ContainerProcess"
+	ContainerCategory    = "ContainerCategory"
 )
 
-func NewProcessObjectOut(aContainer *ObjectOut, aPolicy, aPriority, aCommandLine string) (out *ObjectOut) {
-	out = aContainer
-	out.ObjectType = ContainerProcessType
-	out.ObjectFields[ProcessCommandLine] = aCommandLine
-	out.ObjectFields[SchedulingPolicy] = aPolicy
-	out.ObjectFields[SchedulingPriority] = aPriority
-	return out
-}
-
-func NewContainerObjectOut(aContainer *provider.Container, aReason string, isCompliant bool) (out *ObjectOut) {
-	return NewContainerObjectOutBase(aContainer.Namespace, aContainer.Podname, aContainer.Name, aReason, isCompliant)
+func (obj *ReportObject) SetContainerProcessValues(aPolicy, aPriority, aCommandLine string) *ReportObject {
```

Comment:
Since `obj` is already a pointer, we don't need to return the `*ReportObject`.

Reply:
I agree but the reason why we need the return value in this case is to be able to daisy chain calls to similar functions and eventually assign the result to a variable.